<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1585</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>1221</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddColumn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter une colonne</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteColumn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer une colonne</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddRow">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter une ligne</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteRow">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer une ligne</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveData">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enregistrer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadData">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>la fiche</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1585</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
